libs/src/services/contracts/ProviderSelection.js

```diff
@@ -0,0 +1,79 @@
+
+const Web3 = require('web3')
+
+const ServiceSelection = require('../../service-selection/ServiceSelection')
+const EthWeb3Manager = require('../../../src/services/ethWeb3Manager/index')
+
+/**
+ * This class provides the logic in selecting the proper gateway
+ */
+// TODO: use env vars idk how to use env vars lol
+const gatewayProvidersList = ['https://poa-gateway.audius.co', 'https://core.poa.network']
+
+class ProviderSelection extends ServiceSelection {
+  constructor (whitelist = gatewayProvidersList) {
+    super({
+      whitelist,
```

Comment:
as mentioned before, you shouldn't need a whiltelist. the whitlelist is only useful when you're selecting a subset of what getServices returns, but in this case, you want everything right?

Reply:
yes you're right